docs/fos_http_cache.md

```diff
@@ -0,0 +1,86 @@
+# FOSHttpCacheBundle usage in eZ
+
+Introduced with eZ Publish 5.4, [FOSHttpCacheBundle][fos] is supported by eZ Platform, and covers the following
+features:
+
+* Tags purging/baning (since version 1.8)
```

Comment:
Given that this is now handled in an external package, maybe we can move that down to 1.7 ? I'm gonna test how it behaves with it.